### PR TITLE
keybind tweaks

### PR DESCRIPTION
**Pull request by quardbreak**
## Description of changes
In a few words:
* Removed carbon category.
* Added mob and living category.
* Variable documentation.
* Re-implemented classic hotkeys which I still don't know how they work but.
* Moved button which resets all keybinds up in control preferences
* None -> Unbound. Remind me to replace it with define later.
* New keybinds:
	* LOOC - Unbound.
	* F5 - Msay
	* ShiftF5 - Asay
	* Tab - Now can be used as key. Currently used for hotkey mode switch because I needed to add more features.
	* Screenshot is Unbound by default.
	* G - Give for humans.
	* Space - turns throw mode on while you hold it.

One day byond become drunk and split `auto` argument for `.screenshot` as separate command `.auto`. Auto one don't ask directory location from you to save a screenshot. Probably it defined by default by byond with F2 or ShiftF2 key. 

## Why and what will this PR improve
Keybinds are still meh but slowly heals self.

## Authorship
me me me

### CODE REVIEW

File: code/modules/client/preference_setup/controls/01_keybindings.dm [hunk 1]
```diff
@@ -23,7 +24,7 @@
 		if(length(user_binds[kb.name]))
 			continue // key is unbound and or bound to something
 		var/addedbind = FALSE
-		if(hotkeys)
+		if(client.prefs?.hotkeys)
```
Comment (afterthought2) [inline code/modules/client/preference_setup/controls/01_keybindings.dm]: Why?

**Review by afterthought2 — COMMENTED** [review]
Probably need to test.

File: code/modules/client/preference_setup/controls/01_keybindings.dm (hunk 1 above)
Comment (quardbreak) [reply code/modules/client/preference_setup/controls/01_keybindings.dm]: Idk I running on this place twice and probably mass-replaced this.

**Review by MistakeNot4892 — APPROVED** [review]
Let's give it a go.

**Pull request merged**